lib/kafka_ex/utils/murmur.ex

```diff
@@ -47,7 +47,7 @@ defmodule KafkaEx.Utils.Murmur do
     signed
   end
 
-  # Unsigned bitwise right shift on 32 bits
+  # Unsigned Bitwise right shift on 32 bits
```

Comment:
This is referring to bitwise as in the technical term, not the Bitwise Elixir module.
```suggestion
  # Unsigned bitwise right shift on 32 bits
```